buildSrc/src/main/kotlin/Globals.kt

```diff
@@ -1,3 +1,4 @@
 object Globals {
+    const val artifactPrefix = "datamaintain-"
```

Comment:
What is the purpose of _artefactPrefix_ ? You does not seem to use it in _build.gradle.kts_